### PR TITLE
Add known issue - unavailable vmdp

### PR DESCRIPTION
**Pull request by FrankYang0529**
ref: https://github.com/harvester/harvester/issues/4534

### CODE REVIEW

File: docs/upgrade/v1-1-2-to-v1-2-0.md [hunk 1]
```diff
@@ -398,3 +398,12 @@ If an upgrade is stuck in an `Upgrading System Service` state for an extended pe
   - https://github.com/harvester/harvester/issues/3863#issuecomment-1539681311
 
 ---
+
+### 8. The `registry.suse.com/harvester-beta/vmdp:latest` is not available in airgapped environment
```
Comment (bk201) [inline docs/upgrade/v1-1-2-to-v1-2-0.md:402]:
```suggestion
### 8. The `registry.suse.com/harvester-beta/vmdp:latest` image is not available in airgapped environment
```

Comment (FrankYang0529) [reply docs/upgrade/v1-1-2-to-v1-2-0.md]: Updated it. Thanks.

File: docs/upgrade/v1-1-2-to-v1-2-0.md [hunk 2]
```diff
@@ -398,3 +398,12 @@ If an upgrade is stuck in an `Upgrading System Service` state for an extended pe
   - https://github.com/harvester/harvester/issues/3863#issuecomment-1539681311
 
 ---
+
+### 8. The `registry.suse.com/harvester-beta/vmdp:latest` image is not available in airgapped environment
+
+The harvester hasn't packaged `registry.suse.com/harvester-beta/vmdp:latest` in the ISO since v1.1.0. For Windows VMs before v1.1.0, they used it as a container disk. However, the kubelet may remove old images to free bytes. If the image has been removed, Windows VMs can't get it in the airgapped environment. Users can fix the issue by changing the image to `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` and restarting Windows VMs.
```
Comment (LucasSaintarbor) [inline docs/upgrade/v1-1-2-to-v1-2-0.md:404]:
```suggestion
Harvester does not package the `registry.suse.com/harvester-beta/vmdp:latest` image in the ISO file as of v1.1.0. For Windows VMs before v1.1.0, they used this image as a container disk. However, kubelet may remove old images to free up bytes. Windows VMs can't access an air-gapped environment when this image is removed. You can fix this issue by changing the image to `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` and restarting the Windows VMs.
```

File: versioned_docs/version-v1.2/vm/create-windows-vm.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ The `bootOrder` values need to be set with the installation image first. If you
 3. The **third volume** is a `Container` with the following values:
     1. `Name`: The value `virtio-container-disk` is set by default. You can keep it or change it.
     2. `Type`: Select `cd-rom`.
-    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.3` is set by default. It's recommended you don't change it.
+    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. It's recommended you don't change it.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.2/vm/create-windows-vm.md:75]:
```suggestion
    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. We recommend not changing this value. 
```

File: versioned_docs/version-v1.2/vm/create-windows-vm.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ The `bootOrder` values need to be set with the installation image first. If you
...
     4. `Bus`: The value `SATA` is set by default. It's recommended you don't change it.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.2/vm/create-windows-vm.md:76]:
```suggestion
    4. `Bus`: The value `SATA` is set by default. We recommend not changing this value. 
```

File: docs/vm/create-windows-vm.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ The `bootOrder` values need to be set with the installation image first. If you
 3. The **third volume** is a `Container` with the following values:
     1. `Name`: The value `virtio-container-disk` is set by default. You can keep it or change it.
     2. `Type`: Select `cd-rom`.
-    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.3` is set by default. It's recommended you don't change it.
+    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. It's recommended you don't change it.
```
Comment (LucasSaintarbor) [inline docs/vm/create-windows-vm.md:75]:
```suggestion
    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. We recommend not changing this value.
```

File: docs/vm/create-windows-vm.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ The `bootOrder` values need to be set with the installation image first. If you
...
     4. `Bus`: The value `SATA` is set by default. It's recommended you don't change it.
```
Comment (LucasSaintarbor) [inline docs/vm/create-windows-vm.md:76]:
```suggestion
    4. `Bus`: The value `SATA` is set by default. We recommend not changing this value.
```

File: versioned_docs/version-v1.2/upgrade/v1-1-2-to-v1-2-0.md [hunk 7]
```diff
@@ -398,3 +398,12 @@ If an upgrade is stuck in an `Upgrading System Service` state for an extended pe
   - https://github.com/harvester/harvester/issues/3863#issuecomment-1539681311
 
 ---
+
+### 8. The `registry.suse.com/harvester-beta/vmdp:latest` image is not available in airgapped environment
+
+The harvester hasn't packaged `registry.suse.com/harvester-beta/vmdp:latest` in the ISO since v1.1.0. For Windows VMs before v1.1.0, they used it as a container disk. However, the kubelet may remove old images to free bytes. If the image has been removed, Windows VMs can't get it in the airgapped environment. Users can fix the issue by changing the image to `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` and restarting Windows VMs.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.2/upgrade/v1-1-2-to-v1-2-0.md:404]:
```suggestion
Harvester does not package the `registry.suse.com/harvester-beta/vmdp:latest` image in the ISO file as of v1.1.0. For Windows VMs before v1.1.0, they used this image as a container disk. However, kubelet may remove old images to free up bytes. Windows VMs can't access an air-gapped environment when this image is removed. You can fix this issue by changing the image to `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` and restarting the Windows VMs.
```

**Review by LucasSaintarbor — APPROVED** [review]
Small changes and NITs. LGTM, thanks

File: docs/upgrade/v1-1-2-to-v1-2-0.md (hunk 2 above)
Comment (FrankYang0529) [reply docs/upgrade/v1-1-2-to-v1-2-0.md]: Updated it. Thank you.

File: versioned_docs/version-v1.2/vm/create-windows-vm.md (hunk 3 above)
Comment (FrankYang0529) [reply versioned_docs/version-v1.2/vm/create-windows-vm.md]: Updated it. Thank you.

File: versioned_docs/version-v1.2/vm/create-windows-vm.md (hunk 4 above)
Comment (FrankYang0529) [reply versioned_docs/version-v1.2/vm/create-windows-vm.md]: Updated it. Thank you.

File: docs/vm/create-windows-vm.md (hunk 5 above)
Comment (FrankYang0529) [reply docs/vm/create-windows-vm.md]: Updated it. Thank you.

File: docs/vm/create-windows-vm.md (hunk 6 above)
Comment (FrankYang0529) [reply docs/vm/create-windows-vm.md]: Updated it. Thank you.

File: versioned_docs/version-v1.2/upgrade/v1-1-2-to-v1-2-0.md (hunk 7 above)
Comment (FrankYang0529) [reply versioned_docs/version-v1.2/upgrade/v1-1-2-to-v1-2-0.md]: Updated it. Thank you.

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**